modules/kubernetes/pages/k8s-operator/cluster-operations.adoc

```diff
@@ -80,4 +80,36 @@ To delete the PVCs created with a cluster, run the following command:
 [.wrap,console]
 ----
 $ kubectl delete pvc  -l app=<cluster_name> -n <namespace_name>
-----
\ No newline at end of file
+----
+
+
+[#_shrink_expand_cluster]
+== Shrink or expand a cluster
+Run the following commands to change the size of a cluster.
+Replace `<cluster_name>` with the name of the cluster, and `<namespace_name>` with the namespace where the cluster is provisioned:
+
+[.wrap,console]
+----
+$ kubectl tg update --cluster-name <cluster_name> --size <size> --ha <ha_factor> --namespace <namespace_name>
+----
+
+The `--ha` flag to change the High Availability factor is optional.
+
+[CAUTION]
+After shrinking a cluster, delete the PVC of the pods which were deleted.
+Otherwise, executing the expand command in the future will cause an error for the remaining data on the PVC.
+
+Running `kubectl tg update` automatically creates a backup of the cluster CR before the update.
+If the update fails, delete the failed cluster, then create it again from the backup file.
+
+[.wrap, console]
+----
+kubectl apply -f <path_to_cluster_backup> --namespace <namespace_name>
+----
+
+Remember that an expansion operation creates new pods, so in the case of a failed expansion, the newly created pods should be removed.
+
+For example, if expanding from a cluster size of 5 to 6 results in a failed expansion, delete cluster number `5` (cluster numbering begins with `0`).
```

Comment:
Can we include a sample of what the failed expansion will look like in this scenario?